backend/controllers/post.js

```diff
@@ -9,4 +9,21 @@ module.exports = class Post {
       res.status(500).json({ error: error });
     }
   }
+  static async apiSearchAllpost(req, res, next) {
```

Comment:
move to services folder, PostServices.js

Reply:
I did create a function in PostService.js and called the function in the controller but I don't know why the pull request does now showing the latest change I have